CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## Unreleased
+
+* Merges from_content_id and from_publishing_api into one builder method contained
+in LinkedContentItem. The homepage can now have it's linked expanded through
+it's level_one_taxons by using the homepages content_id to form a nested tree from
```

Comment:
Typo: I'd use `homepage's content_id` or preferably `content_id of the homepage`.